go.mod

```diff
@@ -0,0 +1,63 @@
+module github.com/inizio/skygear-server
```

Comment:
Why don't we just keep it as `skygeario/skygear-server`? It avoids many unnecessary changes.